io_scene_nif/modules/nif_export/property/texture/__init__.py

```diff
@@ -58,18 +58,18 @@ def __init__(self):
 
     def _reset_fields(self):
         self.slots = {
-            "Base": None,
-            "Dark": None,
-            "Emit": None,
-            "Glow": None,
-            "Gloss": None,
-            "Specular": None,
-            "Normal": None,
-            "Bump Map": None,
-            "Detail": None,
-            "Decal 0": None,
-            "Decal 1": None,
-            "Decal2": None,
+                "Base": None,
+                "Dark": None,
+                "Detail": None,
+                "Gloss": None,
+                "Glow": None,
+                "Bump Map": None,
+                "Decal 0": None,
```

Comment:
One thing we may need to consider down the line, is that the Decal/Detail slots are a numbered array. 
I gleaned this from @Greatness7 so we can get more info from him at some point.